src/js/backend/BlockchainLink.js

```diff
@@ -289,7 +305,7 @@ export const initBlockchain = async (coinInfo: CoinInfo, postMessage: $ElementTy
     let backend = find(coinInfo.name);
     if (!backend) {
         backend = new Blockchain({
-            coinInfo: customBackends[coinInfo.shortcut] || coinInfo,
+            coinInfo: customBackends[coinInfo.shortcut] || preferredBackends[coinInfo.shortcut] || coinInfo,
```

Comment:
Shouldn't this be `coinInfo: preferredBackends[coinInfo.shortcut] || customBackends[coinInfo.shortcut] || coinInfo,` ?

Reply:
ha, of course it should!
fixed, thx